aws/cloudformation/bootstrap_frontend.sh.erb

```diff
@@ -20,6 +20,11 @@ pushd $CHEF_CACHE
   /opt/chef/bin/chef-client -z -N <%=node_name%> -o 'recipe[cdo-apps::hostname]'
 popd
 
+# Run rbspy for 2 minutes on 3 dashboard worker processes at startup.
+# We're doing this temporarily to help debug our deployment unavailability issue.
+# Find 3 dashboard worker process IDs and feed them to 3 rbspy invocations, in parallel using xargs.
+ps aux | grep "cluster worker" | grep dashboard | head -n 3 | tr --squeeze-repeats ' ' | cut -d ' ' -f 2 | xargs -P 3 -I {} sudo rbspy record --silent --duration 120 --pid {} &
```

Comment:
I think `sudo` is not needed, since user-data script runs as root? I'm not positive though. In any case, for this debugging action it's probably fine if it's been tested and works as-is.